apps/src/code-studio/pd/application_dashboard/detail_view.jsx

```diff
@@ -41,6 +41,10 @@ export default class DetailView extends React.Component {
     });
   }
 
+  componentWillMount() {
+    this.load();
+  }
+
```

Comment:
nit: move `componentWillMount` up before `load`, so all the standard initialization methods are together.